hedera-node/hedera-network-admin-service-impl/build.gradle.kts

```diff
@@ -22,11 +22,12 @@ dependencies {
     javaModuleDependencies {
         annotationProcessor(gav("dagger.compiler"))
 
-        testImplementation(project(":hedera-node:node-app-service-token-impl"))
-        testImplementation(project(":hedera-node:node-app"))
-        testImplementation(testFixtures(project(":hedera-node:node-config")))
-        testImplementation(testFixtures(project(":hedera-node:node-app")))
-        testImplementation(testFixtures(project(":hedera-node:node-app-spi")))
+        testImplementation(project(":app"))
+        testImplementation(project(":app-service-network-admin-impl"))
+        testImplementation(project(":app-service-token-impl"))
```

Comment:
This is wrong. The NetworkAdmin-Service should not require access to Token-Service implementation. If this wasn't just added by accident, we need to investigate and clean this up.

Reply:
This was reported by the dependency analysis we have now – `./gradlew checkAllModuleInfo`. To make sure this is still there, I ran the analysis again and addressed all additional issues it found.

This one is still there. You can do 
`./gradlew :app-service-network-admin-impl:reason --id :app-service-token-impl`
to get information why:

```
Source: test
------------
* Uses 3 classes: com.hedera.node.app.service.token.impl.ReadableAccountStoreImpl, com.hedera.node.app.service.token.impl.ReadableTokenRelationStoreImpl, com.hedera.node.app.service.token.impl.ReadableTokenStoreImpl (implies testImplementation).
```

That should probably be change then.

Reply:
Thanks, I will create an issue.